### PR TITLE
Add Angular 12 sample

### PR DESCRIPTION
**Pull request by jo-arroyo**
Add new sample using a prerelease version of Angular v12.

### CODE REVIEW

File: samples/msal-angular-v2-samples/angular12-sample-app/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Angular12SampleApp
+
+This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 10.0.0.
```
Comment (tnorling) [inline samples/msal-angular-v2-samples/angular12-sample-app/README.md:3]: Should this read version 12?

File: samples/msal-angular-v2-samples/angular12-sample-app/README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+Run `ng build` to build the project. The build artifacts will be stored in the `dist/` directory. Use the `--prod` flag for a production build.
+
+## Running unit tests
+
+Run `ng test` to execute the unit tests via [Karma](https://karma-runner.github.io).
+
+## Running end-to-end tests
+
+Run `ng e2e` to execute the end-to-end tests via [Protractor](http://www.protractortest.org/).
```
Comment (tnorling) [inline samples/msal-angular-v2-samples/angular12-sample-app/README.md:23]: Update with our e2e command?

**Review by tnorling — APPROVED** [review]

**Review by jasonnutter — APPROVED** [review]
Looks good, can you also update the MSAL Angular README to note that v12 is supported?

File: samples/msal-angular-v2-samples/angular12-sample-app/README.md (hunk 1 above)
Comment (jo-arroyo) [reply samples/msal-angular-v2-samples/angular12-sample-app/README.md]: Thanks for the catch. Updated!

File: samples/msal-angular-v2-samples/angular12-sample-app/README.md (hunk 2 above)
Comment (jo-arroyo) [reply samples/msal-angular-v2-samples/angular12-sample-app/README.md]: Makes sense. Updated!

**Review by hectormmg — APPROVED** [review]

**Review by sameerag — APPROVED** [review]

**Pull request merged**